src/site/SiteNavigation.js

```diff
@@ -226,3 +226,36 @@ document.addEventListener("DOMContentLoaded", onDocumentReady);
  * https://gomakethings.com/listening-for-click-events-with-vanilla-javascript/
  */
 document.addEventListener("click", handleDocumentClick, false);
+
+/**
```

Comment:
just curious, why did you choose to put this in the sitenavigation file? I would have expected this to be placed in https://github.com/baltimorecounty/dotgov-javascript/blob/integration/src/site/Site.js or probably the better solution would be to create a file in site called SocialMedia.js (or something like that)?

Reply:
@tmccoy529

Reply:
hahaha it was late and I was trying to get it done and tested. But you are correct, it should be in its own socialmedia file. This has been changed

Reply:
@tmccoy529 looks good, but i still think you need to import it into Site.js?

Reply:
Hahahah look man, reading comprehension isnt my strong suit.....this has now been imported into site.js